packages/light-apps/src/Onboarding/CreateNewAccountScreen.tsx

```diff
@@ -0,0 +1,178 @@
+// Copyright 2018-2019 @paritytech/substrate-light-ui authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import { AddressSummary, ErrorText, FadedText, Input, Modal, NavButton, NavLink, Segment, Stacked } from '@polkadot/ui-components';
+import keyring from '@polkadot/ui-keyring';
+import { mnemonicGenerate, mnemonicToSeed, naclKeypairFromSeed } from '@polkadot/util-crypto';
+
+import FileSaver from 'file-saver';
+import { inject, observer } from 'mobx-react';
+import React from 'react';
+import { RouteComponentProps } from 'react-router-dom';
+
+import { OnboardingStore } from '../stores/onboardingStore';
+
+interface Props extends RouteComponentProps {
+  onboardingStore: OnboardingStore;
+}
+
+type State = {
+  address?: string;
+  error: string | null;
+  mnemonic: string;
+  name: string;
+  password: string;
+};
+
+function generateMnemonic (): string {
+  const mnemonic = mnemonicGenerate();
```

Comment:
`const generateMnemonic = mnemonicGenerate` :). Actually,do we really need a rename here?

Reply:
Yes, sorry about the naming :( it is basically section + function from util to give it some sort of structure there... so it _should_ be the other way around, but is "wrong" for the sake of lib org.